test/WebJobs.Script.Tests/TestScripts/CSharp/ApiHubTableClient/run.csx

```diff
@@ -1,7 +1,7 @@
 ﻿#r "Microsoft.Azure.ApiHub.Sdk"
 
 using System;
-using Microsoft.Azure.ApiHub.Sdk.Table;
+using Microsoft.Azure.ApiHub;
```

Comment:
Was there a namespace change in the SDK? If so, this would be a breaking change.